lib/proxy.js

```diff
@@ -109,11 +107,15 @@ const PodiumProxy = class PodiumProxy {
         return this.#metrics;
     }
 
+    get registry() {
+        return this.#registry;
```

Comment:
Did you get this working with eslint? My attempts at getting class properties working with eslint failed and let to frustration. The only solution seemed to be to use babel so that eslint could parse the code...

Reply:
This is due to that private properties is not fully landed as a spec yet so ESLint does not support it. The only solution to get ESLint to handle this is to use babel which we do here.